ecosystem/typescript/sdk/src/plugins/coin_client.ts

```diff
@@ -54,8 +61,23 @@ export class CoinClient {
       // If this is the first time an account has received the specified coinType,
       // and this is set to false, the transaction would fail.
       createReceiverIfMissing?: boolean;
+      // fungible asset address if you wish to transfer a fungible asset
+      assetAddress?: MaybeHexString;
     },
   ): Promise<string> {
+    if (extraArgs?.assetAddress) {
```

Comment:
is this too much? what do you think @hariria as someone who worked with this class recently